airbyte-integrations/connectors/source-mysql/src/main/resources/spec.json

```diff
@@ -42,6 +42,13 @@
         "type": "string",
         "order": 5
       },
+      "ssl": {
+        "title": "SSL Connection",
+        "description": "Encrypt data using SSL.",
+        "type": "boolean",
+        "default": true,
+        "order": 7
+      },
```

Comment:
Nitpick. Since this property has `order` 7, can you move it after `replication_method` (whose `order` is 6)?

Reply:
Updated, thanks!